### DISCUSSION TITLE
Installazione interrutori lc400C Bticino Conbee II

### DISCUSSION THREAD

**Discussion by cristianogregnanin** [indomus/forum#1395] [ZigBee, Z-Wave, Thread, Matter]
Ciao a tutti,
ho installato HomeAssistant Core e la chiavetta ConBee II.
Sto usando i frutti Bticino L4003C.
Quando nell'app Phoscon provo a ricercare i frutti è molto instabile, su 6 frutti a volte ne trova 5 a volte nemmeno 1. A volte restano i led rossi accesi.. Il comportamento è talmente casuale che non riesco a fare un adeguato debug. Ho aggiornato tutti i firmware di ConBee e HomeAssistant e mi sono assicurato che la chiavetta usb ConBee II sia vicino ( meno di 5 mentri) ai 6 interruttori connessi, ma niente da fare.
Qualcuno ha avuto problemi simili?

**Comment by dkdkusit** [comment]
Ciao,
mi aggangio a questa discussione perché anche io vorrei capire se ConBee II è compatibile con i deviatori Bticino.
News?

**Reply by cristianogregnanin** [reply]
Ciao è compatibile, ho circa una decina di Bticino L4003C. In fase di ricerca è stato difficile perché era instabile ed aveva comportamenti inattesi. Ad un certo punto ha iniziato ha funzionare.. sicuramente conBee II non è un prodotto professionale, (non lo installerei mai in casa di un cliente), però per smanettare non è male.

**Reply by KC-inDomus** [reply]
scusa, alternativamente come coordinator ZigBee quale sarebbe un "prodotto professionale"?

@dkdkusit non è tanto il coordinator HW ad esserlo, quanto semmai il BRIDGE/Gateway software che lo usa.

**Reply by snakuzzo** [reply]
Certo che con questa storia del "professionale"... :D

**Reply by cristianogregnanin** [reply]
Conosco diverse persone a cui la chiavetta USB si è bruciata, inoltre il software di pairing non è impeccabile. Personalmente ho installato il gateway BTicino e il pairing è stato immediato. Non installerei mai a casa o in azienda da un cliente un prodotto dove mi chiamano alle 11 di sera dicendomi che ha smesso di funzionare.. per professionale intendo questo.

**Reply by KC-inDomus** [reply]
ecco, me ne citi una "professionale"?

**Reply by dkdkusit** [reply]
> Certo che con questa storia del "professionale"... :D

Beh io faccio un altro lavoro, sempre in campo IT, molto spesso (non sempre, per onor del vero) il professionale è quello che costa di più... su cui il vendor può spillare sangue al consumatore finale o all'azienda per il installazione e supporto post-go live.
Insomma una versione 2.0 del Baffo da Crema per vendere prodotti.

Ma tornando al tema: avete provato i deviatori/interruttori Bticino con Echo Plus? Sapete se c'è compatibilità?
Altra e ultima domanda, anche se OT: vorrei installare dei sensori in box e vorrei iniziare ad usare Zigbee, qual è il miglior compromesso? esiste una soluzione "indipendente" dal PC/Raspberry/altro?
Grazie.

**Reply by snakuzzo** [reply]
> Beh io faccio un altro lavoro, sempre in campo IT, molto spesso (non sempre, per onor del vero) il professionale è quello che costa di più

ma infatti la mia risposta era ironica e mi riferivo alla tua risposta dove hai scritto _**"sicuramente conBee II non è un prodotto professionale"**_ , che sinceramente mi sembra una cavolata. Quale sarebbe il prodotto che tu intendi "pofessionale" ?



> Ma tornando al tema: avete provato i deviatori/interruttori Bticino con Echo Plus? Sapete se c'è compatibilità?

Personalmente non li conosco ma, come ti è stato già detto, dipende dal software che gestisce il coordinator che hai intenzione di utilizzare. Zigbee è uno standard "quasi sempre" rispettato...quindi in linea di massima c'è sempre compatibilità. Poi sta allo sviluppatore lato software supportare determinati device, o quantomeno rendere "user friendly" l'utilizzo coi propri hub personali.

> Altra e ultima domanda, anche se OT: vorrei installare dei sensori in box e vorrei iniziare ad usare Zigbee, qual è il miglior compromesso? esiste una soluzione "indipendente" dal PC/Raspberry/altro?

Le soluzioni sono N. Ad oggi il miglior rapporto qualità prezzo è quello del dongle [Sonoff Zigbee 3.0](https://indomus.it/news/hot-itead-sonoff-presenta-il-proprio-coordinator-dongle-usb-zigbee-3-0-compatibile-zigbee2mqtt/). Non sei completamente indipendente, ma potresti risolvere semplicemente collegando la USB ad un RPI Zero W su cui installare zigbee2mqtt o magari solo ser2net per renderla disponibile via TCP. In alternativa hai la soluzione [Z-Bee Duo](https://indomus.it/?s=z-bee+duo) che con un piccolo esp-01 te la rende completamente indipendente e disponibile via TCP

**Reply by dkdkusit** [reply]
> > Beh io faccio un altro lavoro, sempre in campo IT, molto spesso (non sempre, per onor del vero) il professionale è quello che costa di più
> 
> ma infatti la mia risposta era ironica e mi riferivo alla tua risposta dove hai scritto _**"sicuramente conBee II non è un prodotto professionale"**_ , che sinceramente mi sembra una cavolata. Quale sarebbe il prodotto che tu intendi "pofessionale" ?
> 
No no... non sono io ad aver indicato prodotti "professionali"! Io sperimento, con tutti i limiti del caso, qualsiasi cosa home made esista... infatti mi intriga molto Z-Bee Duo che, ammetto, avevo ignorato completamente.

**Reply by KC-inDomus** [reply]
chiedo nuovamente quale possa dunque essere una soluzione professionale, se ConBee non la è :)
